### PR TITLE
[Metrics] Import and generate protos

### PR DESCRIPTION
**Pull request by ncooke3**
### Context
- Import and sanitize the client metrics proto from g3
- Generate the proto header and implementation

### CODE REVIEW

File: GoogleDataTransport/GDTCCTLibrary/Protogen/nanopb/client_metrics.nanopb.c [hunk 1]
```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2019 Google
```
Comment (ncooke3) [inline GoogleDataTransport/GDTCCTLibrary/Protogen/nanopb/client_metrics.nanopb.c:2]:
```suggestion
 * Copyright 2022 Google
```

File: GoogleDataTransport/GDTCCTLibrary/Protogen/nanopb/client_metrics.nanopb.h [hunk 2]
```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2019 Google
```
Comment (ncooke3) [inline GoogleDataTransport/GDTCCTLibrary/Protogen/nanopb/client_metrics.nanopb.h:2]:
```suggestion
 * Copyright 2022 Google
```

File: GoogleDataTransport/GDTCCTLibrary/Protogen/nanopb/client_metrics.nanopb.c [hunk 3]
```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2022 Google
```
Comment (paulb777) [inline GoogleDataTransport/GDTCCTLibrary/Protogen/nanopb/client_metrics.nanopb.c:2]: LLC here and below

**Review by ncooke3 — COMMENTED** [review]
[skip ci] don't forget the LLC


edit: Thought i was making a commit in web editor but actually made a comment

File: GoogleDataTransport/GDTCCTLibrary/Protogen/nanopb/client_metrics.nanopb.c [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Automatically generated nanopb constant definitions */
+/* Generated by nanopb-0.3.9.8 */
```
Comment (paulb777) [inline GoogleDataTransport/GDTCCTLibrary/Protogen/nanopb/client_metrics.nanopb.c:18]: Consider regenning with the current 0.3.9.9. Should be exactly the same - so not necessary if too painful.

**Review by paulb777 — APPROVED** [review]

File: GoogleDataTransport/GDTCCTLibrary/Protogen/nanopb/client_metrics.nanopb.c (hunk 3 above)
Comment (ncooke3) [reply GoogleDataTransport/GDTCCTLibrary/Protogen/nanopb/client_metrics.nanopb.c]: Done

File: GoogleDataTransport/GDTCCTLibrary/Protogen/nanopb/client_metrics.nanopb.c (hunk 4 above)
Comment (ncooke3) [reply GoogleDataTransport/GDTCCTLibrary/Protogen/nanopb/client_metrics.nanopb.c]: Done

**Comment by ncooke3** [comment]
Okay, this PR is good to go. Merging soon.

**Review by chliangGoogle — APPROVED** [review]

**Pull request merged**